client/js/components/statement/voter/StatementVoter.vue

```diff
@@ -392,6 +399,13 @@ export default {
         active: true
       }
       this.updating = false
+    },
+
+    showUpdateForm (index) {
+      for (const key in this.formFields) {
+        this.formFields[key] = this.getVoters[index][key]
+      }
```

Comment:
nitpick: I'd move this to its own function, e.g. `setFormFields`, call it here and add an empty line below

Reply:
I think thats a bit too much. I left it like it is.